docs/getting_started.rst

```diff
@@ -14,13 +14,23 @@ Install with Conda
 ------------------
 .. _install-with-conda:
 
-`conda <http://www.continuum.io/blog/conda>`_ is a python package manager built for scientific python. Unlike ``easy_install`` or ``pip``, it handles binaries and binary dependencies, which are critical for most scientific workflows. If you're a ``conda`` user, you can install OpenPathSampling by adding the omnia channel. If you're not a conda user, you should look into it. ::
+We recommend using `conda` to install OpenPathSampling.  `conda
+<http://www.continuum.io/blog/conda>`_ is a python package manager built for
+scientific python. Unlike ``easy_install`` or ``pip``, it handles binaries
+and binary dependencies, which are critical for most scientific workflows.
+If you're a ``conda`` user, you can install OpenPathSampling by adding the
+omnia channel. ::
 
-To install OpenPathSampling with conda, use the following commands ::
+To install the most recent release of OpenPathSampling with conda, use the
+following commands ::
 
   $ conda config --add channels http://conda.binstar.org/omnia
```

Comment:
> Are you sure you don't want to create a non-dev version, even if it's just a preview release?

I have no opposition to this, but we hadn't done it. I'm just trying to make the docs reflect reality.

> Also, what's up with no py3.x support?

See #70. We decided this wasn't a priority for 1.0, but will be done soon after.

> It's better in general to add the channel (`conda config --add channels omnia`), but that's a bit more advanced than the one-liner needed to install.

I don't think a 2-line install is ***that*** much worse than a 1-line install, especially if it makes updating easier for the complete novice (which is pretty important for the pre-release versions). I'll switch it to just `--add channels omnia`.

> Also, you're missing some dependencies in your conda `meta.yaml`.

Added `matplotlib`. @jhprinz : Can we *use* the conda install for OPS on Travis instead of this separate `conda-requirements` file? That would warn us of problems like that. We would also need to find a way to distinguish between required and recommended package -- something we need to do in any case, to update the non-conda install instructions.